docs/CreateEKSClusterForScalarDB.md

```diff
@@ -1,4 +1,6 @@
-# Guidelines for creating an EKS cluster for ScalarDB Server
+# (Deprecated) Guidelines for creating an EKS cluster for ScalarDB Server
+
+**Note:** ScalarDB Server is now deprecated. Please use [ScalarDB Cluster](./ManualDeploymentGuideScalarDBClusterOnEKS.md) instead.
```

Comment:
ScalarDB Server is now deprecated. So, I added notes for that in existing documents for ScalarDB Server.